evap/staff/tests/test_views.py

```diff
@@ -108,6 +108,79 @@ def test_view_list_sorting(self):
         self.assertLess(position_course1, position_course2)
 
 
+class TestSemesterImportView(ViewTest):
+    url = "/staff/semester/1/import"
+    test_users = ["staff"]
+    filename_valid = os.path.join(settings.BASE_DIR, "staff/fixtures/test_enrollment_data.xls")
+    filename_invalid = os.path.join(settings.BASE_DIR, "staff/fixtures/invalid_enrollment_data.xls")
+
+    @classmethod
+    def setUpTestData(cls):
+        semester = mommy.make(Semester, pk=1)
+        mommy.make(UserProfile, username="staff", groups=[Group.objects.get(name="Staff")])
+        cls.course = mommy.make(Course, pk=1, semester=semester)
```

Comment:
if you don't depend on the primary keys being a specific value, don't set them here.

i also saw that in @bjrne's PR. is that copied from somewhere? if yes, could you fix it there as well?